### ISSUE TITLE
Cannot setup/init on git-submodule-repo because of non-existent .git/config

### ISSUE THREAD

**Issue by joehannes** [semantic-release/cli#360]
## BUG / Feature-Request
Hi there :-)
Am new to semantic-release-cli, so maybe I'm not getting it, but a quick google led nowhere.

### Situation/Reproduce BUG:
Trying to init via `semantic-release-cli setup` gives:
```
ERR! semantic-release Could not get repository url. Please create/add the repository.
ERR! semantic-release Error: ENOTDIR: not a directory, open './.git/config'
```
To reproduce simply try to init semantic-release-cli on a git-submodule dir.
The .git/config doesn't exist since a submodule has a .git file instead of a .git dir.
Example: My `.git file contents`: `gitdir: ../.git/modules/react-fp`

It seems to me that basically semantic-release-cli cannot deal with git-submodules.
But I want ... hh

So maybe this is rather a feature-request, though methinks this could be a nice addition to functionality at very low cost.

Cheers, joehannes

**Comment by mprinc** [comment]
Yes, It is the same problem for me with submodules within a git repo. Did you @joehannes find any workaround?

**Comment by joehannes** [comment]
> Yes, It is the same problem for me with submodules within a git repo. Did you @joehannes find any workaround?

I'm afraid I don't remember for sure, but I think I gave up on this ... sorry :-) @mprinc
